app.js

```diff
@@ -21,6 +24,21 @@ app.get('/js/main.js', (req, res) => {
   });
 });
 
+app.get('/js/login.js', (req, res) => {
```

Comment:
Is this idiomatic? I'm not too familiar with how one configures local javascript.